repair_refurbish/views/repair_view.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_repair_order_tree" model="ir.ui.view">
+        <field name="name">repair.order.tree</field>
+        <field name="model">repair.order</field>
+        <field name="inherit_id" ref="repair.view_repair_order_tree" />
+        <field name="arch" type="xml">
+            <field name="product_id" position="after">
+                <field name="refurbish_product_id" />
+            </field>
+        </field>
+    </record>
+
+    <record id="view_repair_order_form" model="ir.ui.view">
+        <field name="name">repair.order.form</field>
+        <field name="model">repair.order</field>
+        <field name="inherit_id" ref="repair.view_repair_order_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//group[@name='locations']/field[@name='location_id']"
+                position="after"
+            >
+                <field
+                    name="location_dest_id"
+                    invisible="not to_refurbish"
+                    required="to_refurbish"
+                />
+            </xpath>
+            <xpath expr="//group[1]" position="inside">
+                <group>
+                    <field name="to_refurbish" />
+                </group>
+                <group invisible="not to_refurbish">
+                    <field name="refurbish_product_id" required="to_refurbish" />
+                    <field
+                        name="refurbish_lot_id"
+                        domain="[('product_id', '=', refurbish_product_id)]"
+                        context="{'default_product_id': refurbish_product_id, 'default_company_id': company_id}"
+                        groups="stock.group_production_lot"
+                    />
+                    <field name="refurbish_location_dest_id" required="to_refurbish" />
```

Comment:
this should be restricted to `groups="stock.group_stock_multi_locations"`

Reply:
Update, Thanks